tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -15406,3 +15406,200 @@ TEST_F(ZoneUnitarySysTest, UnitarySystemModel_MultiSpeedDXCoilsDirectSolutionTes
      EXPECT_NEAR(thisSys->m_SpeedRatio, 0.228062, 0.02);
      EXPECT_NEAR(sensOut, -11998.0, 210.0);
 }
+
+TEST_F(ZoneUnitarySysTest, UnitarySystemModel_FuelHeatCoilStptNodeTest)
```

Comment:
The new unit test doesn't build. It needs to be brought up to speed with all the latest global data changes.